edx_sga/sga.py

```diff
@@ -443,22 +511,38 @@ def staff_download_annotated(self, request, suffix=''):
             state['annotated_filename']
         )
 
-    def download(self, path, mimetype, filename):
-        BLOCK_SIZE = (1 << 10) * 8  # 8kb
-        file = default_storage.open(path)
-        app_iter = iter(partial(file.read, BLOCK_SIZE), '')
+    def download(self, path, mime_type, filename):
+        """
+        Download file
+        :param path:
+        :param mime_type:
+        :param filename:
+        """
+        block_size = (1 << 10) * 8  # 8kb
```

Comment:
any particular reason of not using a well-known "1024"? (1 << 10 ) will require computation and is cryptic at the same time.